src/components/Deck.js

```diff
@@ -0,0 +1,14 @@
+import { cards } from '../utils/utils';
+
+const Deck = () => {
+    return cards.map((card) => {
```

Comment:
Eu acho que vc pode receber o cards por parâmetro ao invés de ser uma dependência externa. Isso vai facilitar bastante, principalmente na hora de testar.

Vc pode ter uma dependência de um componente (como vc fez com o Deck no App.js) ou de uma função, mas variáveis geralmente são passadas como propriedades para dentro do componente. Algo assim: const Deck = ({ cards }) => {, ai na hora de chamar o componente no App.js vc vai fazer algo assim: <Deck cards={chama_funcao_que_retorna_cards} />.

Vc pode ler mais sobre aqui https://pt-br.reactjs.org/docs/components-and-props.html.

Reply:
Faltou uma explicação ali depois do ":"?